src/quantum/azext_quantum/_help.py

```diff
@@ -283,3 +283,39 @@
         text: |-
             az quantum workspace show -g MyResourceGroup -w MyWorkspace
 """
+
+helps['quantum workspace update'] = """
+    type: command
+    short-summary: Enable or disable Azure Quantum workspace api keys.
+    examples:
+      - name: Enable a provided Azure Quantum workspace api keys.
+        text: |-
+            az quantum workspace update --enable-api-key True
+      - name: Disable a provided Azure Quantum workspace api keys.
+        text: |-
+            az quantum workspace update --enable-api-key False
+"""
+
+helps['quantum workspace keys list'] = """
+    type: command
+    short-summary: Display Azure Quantum workspace api keys.
```

Comment:
List api keys  for the given (or current) Azure Quantum workspace.

Reply:
Let's try to word it consistently with existing doc  https://learn.microsoft.com/en-us/cli/azure/quantum/workspace?view=azure-cli-latest